### DISCUSSION TITLE
Help Getting Google Calendar to Display

### DISCUSSION THREAD

**Discussion by edlovereze3** [MMRIZE/MMM-CalendarExt3#144] [Q&A] [answered]
Hello,

I apologize but I am extremely new to this and am trying to set up my calendar to show my google calendar. I have set up my standard "calendar" module and it shows properly on my MagicMirror but I cannot figure out how to make CalendarExt3 show the events. I can get the blank calendar to work but I cannot get it to pull in my calendar module I already set up. I added a name to my calendar and put that name in the code after calendarSet but I cannot figure it out. I am adding a picture that should be close to my code. Unfortunately I am away from my Pi so i cannot pull exact code but is something missing to call back to the calendar module?

![image](https://github.com/MMRIZE/MMM-CalendarExt3/assets/167540853/20eb3564-0a4d-4efc-ad7e-9cce99d971c4)

**Comment by edlovereze3** [comment, marked as answer]
Apparently I wasn't waiting long enough. It's working now! Nevermind!

**Reply by eouia** [reply]
I'm happy to hear it.

**Reply by edlovereze3** [reply]
Hey I was hoping I could ask you one thing. It takes like 15 minutes for my Google events to load into the calendar. However, the default calendar module only take a mint or so. Do you know why it might do this?

**Reply by eouia** [reply]
Unless you reassign the value `refreshInterval`, the default refreshing cycle would be 10 minutes. In unlucky case, the changed event would be represented in 1 or 2 cycles (10 or 20 minutes).
